specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json

```diff
@@ -152,6 +152,52 @@
                     "application/json"
                 ]
             }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.InfrastructureInsights.Admin/regionHealths/{location}/alerts/{alertName}/remediate": {
+            "post": {
+                "x-ms-examples": {
+                    "Repairs an alert.": {
+                        "$ref": "./examples/Alert/Repair.json"
+                    }
+                },
+                "tags": [
+                    "Alerts"
+                ],
+                "x-ms-long-running-operation": true,
+                "description": "Repairs an alert.",
+                "operationId": "Alerts_Repair",
+                "parameters": [
+                    {
+                        "$ref": "InfrastructureInsights.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "InfrastructureInsights.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "InfrastructureInsights.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AlertNameParameter"
+                    },
+                    {
+                        "$ref": "InfrastructureInsights.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
```

Comment:
Is there nothing returned as part of the response body?

Reply:
If this is a long running operation please mark it as such.  If the long running operation returns an object please modify the 200 and 202 to return that object, which MUST be an Alert object.

Reply:
When we do the post operation, the response body is empty and the operation tracking URI is in the response headers. So yes, there is nothing returned as a part of the response body.

Reply:
Please add        "x-ms-long-running-operation": true